contrib/scalar/scalar.c

```diff
@@ -133,7 +133,7 @@ static int set_recommended_config(void)
 		{ "receive.autoGC", "false" },
 		{ "reset.quiet", "true" },
 		{ "status.aheadBehind", "false" },
-		{ "config.writeConfigLockTimeoutMS", "150" },
+		{ "core.writeConfigLockTimeoutMS", "150" },
```

Comment:
why isn't this `config.writeLockTimeoutMs`? Is there no `Documentation/config/config.txt`?

Reply:
There is indeed no `Documentation/config/`, and I would not want to start one _just_ for this ;-)